<launch>
	<group ns="fmControllers">
		
		<node pkg="sdu_pichi" name="robot_interface" type="sdu_pichi_node" output="screen" >
			<param name="cmd_vel_ch1_topic" value="/fmSignals/cmd_vel_left"/>
			<param name="cmd_vel_ch2_topic" value="/fmSignals/cmd_vel_right"/>
			<param name="serial_rx_topic" value="/fmData/robot_rx"/>
			<param name="serial_tx_topic" value="/fmData/robot_tx"/>
			<param name="deadman_topic" value="/fmSignals/deadman"/>
			<param name="encoder_ch1_topic" value="/fmInformation/encoder_left"/>
			<param name="encoder_ch2_topic" value="/fmInformation/encoder_right"/>	
			<param name="power_ch1_topic" value="/fmData/power_left"/>
			<param name="power_ch2_topic" value="/fmData/power_right"/>
			<param name="propulsion_module_status_topic" value="/fmInformation/propulsion_module_status"/>
			<param name="propulsion_module_feedback_left_topic" value="/fmInformation/propulsion_module_feedback_left"/>
			<param name="propulsion_module_feedback_right_topic" value="/fmInformation/propulsion_module_feedback_right"/>
			<param name="status_topic" value="/fmData/status"/>	
			<param name="ticks_per_meter_left" value="1285.0"/>
			<param name="ticks_per_meter_right" value="-683.5"/>
			<param name="p_gain" value="10"/>
			<param name="i_gain" value="1"/>
			<param name="d_gain" value="0"/>
			<param name="i_max" value="1"/>
			<param name="max_controller_command" value="400" />
			<param name="closed_loop_operation" value="true"/>        
		</node>
	</group>
</launch>
